### PR TITLE
fix: Error when no theme is given

### PR DESCRIPTION
**Pull request by IagoLast**

### CODE REVIEW

**Review by Claratj — APPROVED** [review]

File: index.html
Comment (alejandraarri) [inline index.html:9]: This change should be reverted. Do you want to wait until we merge my PR? If we don't release the PCI SDK we can wait.

File: packages/pci-sdk-web/src/index.ts
Comment (alejandraarri) [inline packages/pci-sdk-web/src/index.ts:185]: I think we shouldn't set the default theme here, this should be optional. All the default values are set in the iFrame. If the user doesn’t specify any theme, we shouldn’t pass this parameter to the iFrame.

**Pull request merged**
